www/api/class/centreon_configuration_service_severity.class.php

```diff
@@ -0,0 +1,95 @@
+ * General Public License cover the whole combination.
+ *
+ * As a special exception, the copyright holders of this program give Centreon
+ * permission to link this program with independent modules to produce an executable,
+ * regardless of the license terms of these independent modules, and to copy and
+ * distribute the resulting executable under terms of Centreon choice, provided that
+ * Centreon also meet, for each linked independent module, the terms  and conditions
+ * of the license of that module. An independent module is a module which is not
+ * derived from this program. If you modify this program, you may extend this
+ * exception to your version of the program, but you are not obliged to do so. If you
+ * do not wish to do so, delete this exception statement from your version.
+ *
+ * For more information : contact@centreon.com
+ *
+ */
+
+
+require_once _CENTREON_PATH_ . "/www/class/centreonDB.class.php";
+require_once __DIR__ . "/centreon_configuration_objects.class.php";
+
+class CentreonConfigurationServiceSeverity extends CentreonConfigurationObjects
+{
+    /**
+     * CentreonConfigurationServicecategory constructor.
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * @return array
+     * @throws Exception
+     */
+    public function getList()
+    {
+        $queryValues = array();
+
+        // Check for select2 'q' argument
+        if (false !== isset($this->arguments['q'])) {
+            $queryValues['name'] = '%' . (string)$this->arguments['q'] . '%';
+        } else {
+            $queryValues['name'] = '%%';
+        }
+
+        $queryContact = 'SELECT SQL_CALC_FOUND_ROWS DISTINCT sc_id, sc_name FROM service_categories ' .
+            'WHERE sc_name LIKE :name ' .
+            "AND level IS NOT NULL " .
+            'ORDER BY sc_name ';
+
+        if (isset($this->arguments['page_limit']) && isset($this->arguments['page'])) {
+            if (!is_numeric($this->arguments['page']) || !is_numeric($this->arguments['page_limit'])) {
+                throw new \RestBadRequestException('Error, limit must be numerical');
+            }
+            $offset = ($this->arguments['page'] - 1) * $this->arguments['page_limit'];
+            $queryContact .= 'LIMIT :offset,:limit';
+            $queryValues['offset'] = (int)$offset;
+            $queryValues['limit'] = (int)$this->arguments['page_limit'];
+        }
+        $stmt = $this->pearDB->prepare($queryContact);
+        $stmt->bindParam(':name', $queryValues['name'], PDO::PARAM_STR);
+        if (isset($queryValues['offset'])) {
+            $stmt->bindParam(':offset', $queryValues["offset"], PDO::PARAM_INT);
+            $stmt->bindParam(':limit', $queryValues["limit"], PDO::PARAM_INT);
+        }
+        $stmt->execute();
+        $serviceList = array();
+        while ($data = $stmt->fetch()) {
+            $serviceList[] = array('id' => $data['sc_id'], 'text' => $data['sc_name']);
```

Comment:
for better readability, you could make each array key/value on a new line. Like on line 91